src/physics/forceCal/forceCal.cpp

```diff
@@ -0,0 +1,48 @@
+
+#include "physics/forceCal/forceCal.h"
+#include "utils/ArrayUtils.h"
+#include <unordered_map>
+#include <vector>
+
+void force_stroemer_verlet(const Simulation& sim)
```

Comment:
I wouldn't call it `force_stroemer_verlet` as the force doesn't depend on the time descretization but the gravity. So `force_gravity` would be a more appropriate name